project/maps2params/python/maps_to_params_utils.py

```diff
@@ -256,3 +256,32 @@ def is_symmetric(mat, tol=1e-8):
 
 def is_pos_def(mat):
     return np.all(np.linalg.eigvals(mat) > 0)
+
+def calibrate_alm(alms, exp, freq, **nuis_params):
```

Comment:
The `exp` argument is only used by the print statement. I think it will be better to remove it since it is not needed by the code itself and also to move the print statement into the main code i.e. [here](https://github.com/simonsobs/PSpipe/pull/99/files#diff-1497f07c510afeaf56bb31f050517e728b23dc3481275902b478a235286d3752R113)